service/create/launch_configuration.go

```diff
@@ -91,21 +93,23 @@ func (s *Service) createLaunchConfiguration(input launchConfigurationInput) (boo
 		return false, microerror.MaskAny(err)
 	}
 
-	launchConfigName, err := launchConfigurationName(input.cluster, input.prefix)
+	launchConfigName, err := launchConfigurationName(input.cluster, input.prefix, securityGroupID)
```

Comment:
There's a comment in the function's definition that explains that.

Basically, it happened to me that leftover LCs would match, but they had been created with different SGs (which didn't exist anymore).

Admittedly, this might have been a temporary situation. WDYT?